app/modules/sightings/models.py

```diff
@@ -168,3 +168,29 @@ def augment_edm_json(self, edm_json):
             json, err = asset_schema.dump(asset)
             edm_json['assets'].append(json)
         return edm_json
+
+    # pass results-json for sighting.encounters from changes made (e.g. PATCH) and update houston encounters accordingly
+    def rectify_edm_encounters(self, edm_encs_json):
+        log.debug(f' RECTIFY IN {edm_encs_json}')
+        edm_map = {}  # id => version of new results
+        if edm_encs_json:
+            for edm_enc in edm_encs_json:
+                edm_map[edm_enc['id']] = edm_enc
+        log.debug(f' RECTIFY EDM_MAP {edm_map}')
+        # find which have been removed and which updated
+        if self.encounters:
+            for enc in self.encounters:
+                if str(enc.guid) not in edm_map.keys():
+                    log.info(f'houston Encounter {enc.guid} removed')
```

Comment:
worth adding current_user.email to trace?

Reply:
totally agree; this is the kind of thing worth noting _somehow_ ......  except for i would argue a better thing to do here would be to use the - as-yet-developed **audit log** for something like this.  :grimacing:   https://wildme.atlassian.net/browse/PNG-87

i am going to add a TODO for audit log here -- maybe even we should have some sort of codified audit-log-specific TODO?

Reply:
I think that is a great idea. It will be super helpful for whichever future one of us hooks in audit log.